dash/dash-renderer/src/TreeContainer.js

```diff
@@ -476,9 +476,16 @@ class BaseTreeContainer extends Component {
 
     render() {
         const {
+            _dashprivate_error,
             _dashprivate_layout,
             _dashprivate_loadingState,
-            _dashprivate_path
+            _dashprivate_loadingStateHash,
+            _dashprivate_path,
+            _dashprivate_config,
+            _dashprivate_dispatch,
+            _dashprivate_graphs,
+            _dashprivate_loadingMap,
+            ..._dashextra_controlProps
```

Comment:
Look like those variables are not used and fail the linting, maybe disable for that line or use ramda.without to get the rest.

Reply:
@T4rk1n I have modified the code and added a test. Due to my unfamiliarity with dash testing, I am unsure if this test complies with dash's testing rules.

Reply:
@T4rk1n This PR has passed all the checks. Is there anything else that needs to be modified in this PR?